src/wallet/tx_builder.cpp

```diff
@@ -74,17 +75,7 @@ static constexpr T div_ceil(T dividend, T divisor)
 }
 //-------------------------------------------------------------------------------------------------------------------
 //-------------------------------------------------------------------------------------------------------------------
-static epee::misc_utils::auto_scope_leave_caller make_fcmp_obj_freer(const std::vector<uint8_t*> &objs)
-{
-    return epee::misc_utils::create_scope_leave_handler([&objs](){
-        for (uint8_t *obj : objs)
-            if (nullptr != obj)
-                free(obj);
-    });
-}
-//-------------------------------------------------------------------------------------------------------------------
-//-------------------------------------------------------------------------------------------------------------------
-static bool is_transfer_usable_for_input_selection(const wallet2_basic::transfer_details &td,
+static bool is_transfer_usable_for_input_selection(const wallet2::transfer_details &td,
```

Comment:
Just this line should be reverted back to `const wallet2_basic::transfer_details &td,`

Reply:
Good catch